lib/presentation/controllers/login_controller.dart

```diff
@@ -47,31 +49,31 @@ class LoginController extends ControllerBase{
 
   String? passwordValidator(String? value){
     if (_alwaysValidateInvalidCredentials){
-      return 'Invalid credentials';
+      return AppLocalizations.of(Get.context!)!.invalidCredentials;
     }
 
     if (value == null || value == ''){
-      return 'Password must not be empty';
+      return AppLocalizations.of(Get.context!)!.passwordNotEmpty;
     }
 
     if (value.length > 255){
-      return 'Password must not be longer than 255 characters';
```

Comment:
Can't this be somehow parametrized? For example, the translation is "Password must not be longer than $maxPasswordLen characters", and with Intl you can do something like:
```dart
return Intl.message("Password must not be longer than $maxPasswordLen characters",
args: [_maxPasswordLen]);
```
AFAIK

If the max length changes in the future, there will be only one place, that depends on it.

Reply:
Yes, I'll change that